resources/views/livewire/users/list.blade.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+use App\Models\Chirp;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Collection;
+use Livewire\Attributes\On;
+use Livewire\Volt\Component;
+
+new class extends Component {
```

Comment:
We should definitely add pagination to this. I've done some research and the route the Laravel Bootcamp took you down means it's slightly different to what I thought, but still possible:

https://livewire.laravel.com/docs/volt#providing-additional-view-data